packages/sdk/src/client.ts

```diff
@@ -565,84 +586,90 @@ export class Client
 
     private async createUserStream(
         userStreamId: string | Uint8Array,
-        metadata: { spaceId: Uint8Array } | undefined,
+        metadata?: { spaceId?: Uint8Array; appAddress?: Uint8Array },
     ): Promise<ParsedStreamResponse> {
         const userEvents = [
             await makeEvent(
                 this.signerContext,
                 make_UserPayload_Inception({
                     streamId: streamIdAsBytes(userStreamId),
+                    appAddress: metadata?.appAddress ?? bin_fromHexString(''),
```

Comment:
appAddress is optional in the protocol, you shouldn't need `?? bin_fromHexString`